### DISCUSSION TITLE
filtered=true in fileset

### DISCUSSION THREAD

**Discussion by mrekajondrej-anasoft** [eclipse/jkube#3582] [Q&A] [answered]
We are currently using the fabric8-maven-plugin. While migrating to the kubernetes-maven-plugin, we encountered different behavior in the assembly section. Specifically, with fabric8, it is possible to define filtered=true:

`
xml
<fileSet>
  <filtered>true</filtered>
`
As a result, variables with content ${*} are resolved during copying. Is it possible to achieve similar behavior with the kubernetes-maven-plugin?

**Comment by manusa** [comment, marked as answer]
This is not built into the plugin as it was before.
The main reason is that in the past, this was leveraging Maven tooling to achieve the filtering.
Since now JKube provides Gradle, Maven, and standalone ways of processing, the feature was not directly integrated.

This question was asked before and I have a very old demo showcasing how to configure the project to achieve the same result:

https://github.com/marcnuri-demo/eclipse-jkube-complex-assemblies

The code is extremely outdated, but you should be able to use the same procedure in your project.

**Reply by mrekajondrej-anasoft** [reply]
thanks
